[07.11.2024 17:35:43]: Starting!
[07.11.2024 17:35:43]: Accepting a client
[07.11.2024 17:35:43]: Sending the public key (RSA) to the client
[07.11.2024 17:35:43]: SENDING(None): {"code":0,"publicKey":{},"modulus":"ywmuIPBKLo4H0tKUBuJPdilzS6YaAKJ48lvMX56M06TT8qro7YOG6hvYpaw8CuJTJudoneKLVmUsxvGz8cO8ZBPUdN0ke7rYPII3IYDhNjvK0V8EhbrtbH5sWowlL483BRW09KIJPPKUfRy743lpXAyTf33Y1ulEJtPaVEolOM\u002BN7dINGWdISh5iZVSyZwxuEzQ4BK42pa4ii/NM\u002BPhPBVc\u002BplpTNsosKHhWgjPipJlyn4V680tQqQMEUyLL5DPa1i8XQlfoeHQP2YDpoA9OGv8B5uJOL9pIBAqw/r2oDCx4lgX4HUzk41r5IysblAbD/oFQr8qhznnZgvlwl5WYG/zpgsi1MH72PlSagpY2FArihmToRt9\u002B1OBfL/UqgnvAyyZYwE4odncfbVWukA/KNsftSRc8OslgIx3krgzf4Em50aDIU5HzqHdU2STrOyplG\u002BxGU4K2E72H/7BVevr2GZgLczF0qqQhRyGyXKLQn2DIowgPP07EnycIA5TVKEwVitpI0Plu53\u002Bw4nWiE6\u002B1pN5jefoNYiA9a14C4MbU\u002BoU5pBKx09wI2qwF5fqPcx1bivNkP3zUElIK7Is2yaQZNe5gxzy0s8/tl0MyXDxa5tDPljXKwnAY1w75AmkTZrP1jZsnJn5\u002B3eKpt0kBz/jlv7Ss/6Asc/iOizFn0rfc9K0=","exponent":"AQAB"}
[07.11.2024 17:35:43]: Connected sucessfully to the server!
[07.11.2024 17:35:43]: RECEIVED: The received payload is 785 bytes long
[07.11.2024 17:35:43]: RECEIVED: {"code":0,"publicKey":{},"modulus":"ywmuIPBKLo4H0tKUBuJPdilzS6YaAKJ48lvMX56M06TT8qro7YOG6hvYpaw8CuJTJudoneKLVmUsxvGz8cO8ZBPUdN0ke7rYPII3IYDhNjvK0V8EhbrtbH5sWowlL483BRW09KIJPPKUfRy743lpXAyTf33Y1ulEJtPaVEolOM\u002BN7dINGWdISh5iZVSyZwxuEzQ4BK42pa4ii/NM\u002BPhPBVc\u002BplpTNsosKHhWgjPipJlyn4V680tQqQMEUyLL5DPa1i8XQlfoeHQP2YDpoA9OGv8B5uJOL9pIBAqw/r2oDCx4lgX4HUzk41r5IysblAbD/oFQr8qhznnZgvlwl5WYG/zpgsi1MH72PlSagpY2FArihmToRt9\u002B1OBfL/UqgnvAyyZYwE4odncfbVWukA/KNsftSRc8OslgIx3krgzf4Em50aDIU5HzqHdU2STrOyplG\u002BxGU4K2E72H/7BVevr2GZgLczF0qqQhRyGyXKLQn2DIowgPP07EnycIA5TVKEwVitpI0Plu53\u002Bw4nWiE6\u002B1pN5jefoNYiA9a14C4MbU\u002BoU5pBKx09wI2qwF5fqPcx1bivNkP3zUElIK7Is2yaQZNe5gxzy0s8/tl0MyXDxa5tDPljXKwnAY1w75AmkTZrP1jZsnJn5\u002B3eKpt0kBz/jlv7Ss/6Asc/iOizFn0rfc9K0=","exponent":"AQAB"}
[07.11.2024 17:35:43]: RECEIVED: Received opCode ReceiveRSA
[07.11.2024 17:35:43]: Sending aes
[07.11.2024 17:35:44]: ERROR in file Client.cs, in MoveNext(), at line: 140, at column: 13
[07.11.2024 17:35:44]: ERROR: Exception of type 'System.Exception' was thrown.
[07.11.2024 17:35:52]: ERROR in file Server.cs, in MoveNext(), at line: 50, at column: 21
[07.11.2024 17:35:52]: ERROR: An internal WebSocket error occurred. Please see the innerException, if present, for more details. 
[07.11.2024 17:35:52]: Lost the connection with a client
